doc/en/unittest.rst

```diff
@@ -180,3 +180,7 @@ was executed ahead of the ``test_method``.
    to selectively leave away the ``unittest.TestCase`` subclassing, use
    plain asserts and get the unlimited pytest feature set.
 
+Converting from unittest to pytest
+---------------------------------------
+
+If you want to convert your unittest testcases to pytest, there are some helpers. E.g. `unittest2pytest <https://pypi.python.org/pypi/unittest2pytest/>`__, which uses lib2to3 ind introspection for the transformation.
```

Comment:
ind -> and

I also think that should be something like ", there are some helpers like unittest2pytest ...".